oci/config.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+import ConfigParser
+import os
+
+gerrit_default_host = 'gerrit.ovirt.org'
+jenkins_default_host = 'jenkins.ovirt.org'
+
+
+def load():
+    config_dir = os.path.join(os.path.expanduser('~'), '.config/oci.conf')
```

Comment:
We can simplify to:

    os.path.expanduser("~/.config/oci.conf")

See build-artifacts.py system-tests.py